app/views/handlers/content.py

```diff
@@ -34,6 +34,14 @@ def get_context(self):
                 self._current_location.list_item_id,
                 self._questionnaire_store,
             ),
+            "show_individual_response_link": show_individual_response_link(
+                self._current_location, self._schema
+            )
```

Comment:
Currently there's a lot of duplication here.

Given both the hub/interstitial templates already determine visibility based on existence of the url, I would expect the existing individual_response_url function to be updated to factor in the requirements of the interstitial pages, rather than a new variable be created and only used on interstitial templates.

Reply:
Duplication removed, extra variable gone. Just need to add tests now👍.